package/AUTHORS

```diff
@@ -233,9 +233,11 @@ Chronological list of authors
   - Lawson Woods
   - Johannes Stöckelmaier
   - Jenna M. Swarthout Goddard
-  2024
+
+2024
```

Comment:
```suggestion
  2024
```
Just need to revert this back to the previous formatting.

Reply:
@IAlibay all the rest of the years in the Authors file does not have any space except for 2024 which has two spaces, I have aligned the year 2024 with the rest of the years, if you still want me to revert the changes, I will do that.

Reply:
![image](https://github.com/MDAnalysis/mdanalysis/assets/12460125/8c109e02-728b-405e-a35c-c3241d97b899)

Sorry @nataliyah123 I don't think I fully understand what you mean. The image above (of the 2022 entry) is how it should look for 2024. My suggested change should fix things to how they should be.

Reply:
@IAlibay made the changes you suggested in both files. and reverted the changes you asked. I was talking about the spaces as shown in the image below. 
![image](https://github.com/MDAnalysis/mdanalysis/assets/5617255/e883d853-2918-412f-8cb2-a5ef3c178074)